lib/core/src/lib/datatable/components/datatable/datatable.component.scss

```diff
@@ -11,7 +11,7 @@ $data-table-card-padding: var(--theme-headline-line-height) !default;
 $data-table-cell-top: calc($data-table-card-padding / 2);
 $data-table-thumbnail-width: 65px !default;
 $data-table-cell-min-width: 65px !default;
-$data-table-cell-min-width-no-grow: 100px !default;
```

Comment:
this change does not make sense for me... what about other languages that take more than that? German for instance?

Reply:
In order to make a generic fix, we will have to make the cell of the data table, more flexible in terms of height and width both. I will make the changes accordingly and discuss the design changes with Shane Maciak.